mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -7,17 +7,45 @@ CLASS bqp net/minecraft/block/RedstoneWireBlock
 	FIELD f DIRECTION_TO_WIRE_CONNECTION_PROPERTY Ljava/util/Map;
 	FIELD g WIRE_CONNECTIONS_TO_SHAPE [Lcsp;
 	FIELD h wiresGivePower Z
+	FIELD i lateNeighborUpdates Ljava/util/Set;
 	METHOD <init> (Lbml$c;)V
 		ARG 1 settings
+	METHOD a increasePower (ILbvj;)I
+		ARG 1 power
+		ARG 2 state
+	METHOD a connectsTo (Lbgr;Lev;)Z
+		ARG 0 view
+		ARG 1 pos
 	METHOD a getRenderConnectionType (Lbgr;Lev;Lfa;)Lbwo;
 		ARG 1 view
 		ARG 2 pos
+		ARG 3 dir
+	METHOD a updateNeighborsIfWire (Lbhh;Lev;)V
+		ARG 1 world
+		ARG 2 pos
+	METHOD a updateNeighbors (Lbhh;Lev;Lbvj;)Lbvj;
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+	METHOD a connectsTo (Lbvj;Lfa;)Z
+		ARG 0 state
+		ARG 1 dir
 	METHOD b getWireColor (I)I
 		ARG 0 powerLevel
+	METHOD b (Lbgr;Lev;Lfa;)Z
+		ARG 1 view
+		ARG 2 pos
+		ARG 3 dir
+	METHOD b updateHere (Lbhh;Lev;Lbvj;)Lbvj;
```

Comment:
Sounds a bit silly to me

Reply:
I am really struggling with these redstone wire mappings D:

Reply:
What this does, is calculates the current redstone power in the world ignoring power from other redstone dusts, finds 1 minus the highest powered neighbor redstone dust, and sets the power of the redstone dust blockstate at the given world and position to the greater of the two. It also collects a bunch of `blockpos`es into what I've called the `lateNeighborUpdates` set, the calling method calls `updateNeighborsAlways` on that set.

What's less verbose than `recalculatePowerAndCollectNeighorUpdates`...

Reply:
``updatePower``, ``lateNeighborUpdates`` -> ``affectedNeighbors``

Reply:
> What's less verbose than `recalculatePowerAndCollectNeighorUpdates`

`computeWireState`?

Reply:
Doesn't just compute, it *sets* the state too, or so quat says

Reply:
`updateWireSate`?

Reply:
updateLogic or updateWireLogic?